### PR TITLE
feat: add district metrics for 2021

### PR DESCRIPTION
**Pull request by Log234**
This PR adds district metrics data for 2021. The data is needed for https://github.com/Project-Lavinia/Lavinia-client/issues/80 .

### CODE REVIEW

File: Lavinia-api/Lavinia-api.xml [hunk 1]
```diff
@@ -639,13 +639,5 @@
...
             <returns>List of PartyVotes</returns>
         </member>
-        <member name="M:LaviniaApi.Utilities.ModelBuilder.BuildParties(System.Collections.Generic.IEnumerable{LaviniaApi.Utilities.ResultFormat})">
-            <summary>
-            Takes a list of ResultFormat and returns a list of Parties.
-            The returned list contains information about which parties participated in a particular election.
-            </summary>
-            <param name="election">List of ResultFormat</param>
-            <returns>List of Party</returns>
-        </member>
```
Comment (simendag) [inline Lavinia-api/Lavinia-api.xml:649]: This is an unrelated change -- I guess something that happened automatically at some point due to the way the partycode-partyname map works?

Comment (Log234) [reply Lavinia-api/Lavinia-api.xml:649]: Potentially, I did not manually change that file.
